proc-macros/tests/ui/incorrect/sub/sub_unsupported_field.rs

```diff
@@ -4,7 +4,7 @@ use jsonrpsee::proc_macros::rpc;
 #[rpc(client, server)]
 pub trait UnsupportedField {
 	#[subscription(name = "sub", unsubscribe = "unsub", item = u8, magic = true)]
-	async fn sub(&self);
+	async fn sub(&self) -> RpcResult<()>;
```

Comment:
I guess you could `impl IntoSubscriptionResponse` for `()` too (ie no message or error sent back) if you want to allow empty return types.

Reply:
yeah I was thinking the same, why not

Reply:
implemented now